.github/workflows/build.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Install GitVersion
       uses: gittools/actions/gitversion/setup@v0.9.9
       with:
-        versionSpec: "5.6.9"
+        versionSpec: "5.12.0"
```

Comment:
Fine to change this, but this actually controls the version of the spec we want to use. So that's the version scheme that GitVersion uses to create a version based on the git commits in your repo. There's a lot of different versions flying around in this area so can be confusing. 

GitVersion is a tool that computes the version of our code based on the git history (tags etc). The spec version controls the version of the versioning scheme 😂. Then there's the version of the GitHub actions tool that is a wrapper around GitVersion to get it installed in the build pipeline here. That's the version we're interested in here as it controls the specifics of how GitVersion is setup and run during the build.

Reply:
LOL I was shooting in the dark here. I think I get what you mean, reverted some changes. Let me know if this is still incorrect.

Reply:
Hmmm looks like I might have been wrong here. It seems your previous change actually got the build further, so fine to bump the specVersion too. Tbh as long as it builds and the version number generation is sensible then it's fine.

Reply:
Yeah so the specVersion actually specifies the version of the underlying GitVersion executable I think and that's what needs to be updated to work with more recent .NET SDKs.

Reply:
Yeah ideally I'd like to just move this onto the latest .NET SDK.